modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,169 @@ The properties for broker-wide throughput quota balancing are configured at the
++
+[,bash]
 ----
-"kafka_client_group_byte_rate_quota": [
-  {
-    "group_name": "group_1",
-    "clients_prefix": "producer_group_alone_producer",
-    "quota": 10240
-  },
-  {
-    "group_name": "group_2",
-    "clients_prefix": "producer_group_multiple",
-    "quota": 20480
-  },
+rpk cluster config get kafka_client_group_byte_rate_quota
+
+----
+// TODO: Confirm how to retrieve for multiple quota types:
+// kafka_client_group_fetch_byte_rate_quota
+// target_quota_byte_rate
+// target_fetch_quota_byte_rate
+// kafka_admin_topic_api_rate
++
+[,bash,role=no-copy]
+----
+"kafka_client_group_byte_rate_quota": [ 
+  { 
+    "group_name": "group_1", 
+    "clients_prefix": "producer_group_alone_producer", 
+    "quota": 10240 
+  }, 
+  { "group_name": "group_2", 
+    "clients_prefix": "producer_group_multiple", 
+    "quota": 20480 
+  }, 
 ]
 ----
+
+. Each client quota cluster property corresponds to a quota type in Kafka. Check the corresponding `rpk` arguments to use when setting the new quota values:
++
+|===
+| Cluster configuration property | `rpk cluster quotas` arguments
+
+| xref:reference:properties/cluster-properties.adoc#target_quota_byte_rate`[`target_quota_byte_rate`]                      
+| `--default client-id --add producer_byte_rate=<producer-byte-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#target_fetch_quota_byte_rate[`target_fetch_quota_byte_rate`]                
+| `--default client-id --add consumer_byte_rate=<consumer-byte-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#kafka_admin_topic_api_rate[`kafka_admin_topic_api_rate`]                  
+| `--default client-id --add controller_mutation_rate=<controller-mutation-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#kafka_client_group_byte_rate_quota[`kafka_client_group_byte_rate_quota`]          
+| `--name client-id-prefix=<prefix> --add producer_byte_rate=<group-producer-byte-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#kafka_client_group_fetch_byte_rate_quota[`kafka_client_group_fetch_byte_rate_quota`]    
+| `--name client-id-prefix=<prefix> --add consumer_byte_rate=<group-consumer-byte-rate-value>`
+
+|===
++
+The client throughput quotas set through the Kafka API apply per broker, so you need to convert the cluster configuration values that were applied on a per-shard basis. For example, if you set `target_fetch_quota_byte_rate` to 100MBps/shard, and you run Redpanda on 16-core brokers, you can set the new consumer_byte_rate quota to 100 * 16 = 1600MBps.
+
+. Use `rpk cluster quotas alter` to set the corresponding client throughput quotas based on the Kafka API:
++
+[,bash]
+----
+rpk cluster quotas alter --name client-id-prefix=producer_group_alone_producer --add producer_byte_rate=10240 
```

Comment:
@pgellert Is this also an example where the value could be converted from the per-shard setting to per-broker? Just checking because these are the same example values from the cluster config.

Reply:
All of the 5 cluster configs are converted (multiplied by the core count), but anything configured with `rpk cluster quotas alter` is not converted. So it's probably best for the example to also be scaled up to maintain the existing behaviour. Something like this:
```
rpk cluster quotas alter --name client-id-prefix=producer_group_alone_producer --add producer_byte_rate=$((10240*$BROKER_CORE_COUNT))
```